templates/patterns/timeline/pattern-timeline.html.twig

```diff
@@ -26,6 +26,11 @@
     {{ title }}
   </h2>
 {% endif %}
+{% if introduction %}
+  <div id="timeline-introduction">
```

Comment:
Not sure it's a good idea to use a id/class for testing purposes only. Definitely not an ID.

Also, maybe we need an ECL class because i am seeing the text not wrapped in the same class as when i use this same field in a rich text paragraph. And yes, this is in EWCMS with the text format on the field:

<img width="1082" alt="Screenshot 2022-01-27 at 11 46 04" src="https://user-images.githubusercontent.com/5848933/151343990-b9c5193c-947e-4668-9f92-5f053e33a168.png">